files/2da/2da_bgee/clssplab.htm

```diff
@@ -0,0 +1,49 @@
+---
+title: "BG(2):EE 2DA Files : clssplab.2da"
+---
+    <div class="title_main">clssplab.2da</div>
+<br />
+
+For use with <a href="../../../opcodes/bgee.htm#op15">opcode #15</a> and <a href="../../../opcodes/bgee.htm#op44">opcode #44</a>.
+<br /><br />
+
+In IWD:EE we have:<br />
+
+<pre>
+2DA V1.0
+6
+                    DEX		STR
+UNUSED              6		8
+MAGE                6		4
+FIGHTER             6		8
+CLERIC              4		6
+THIEF               8		6
+BARD                8		6
+PALADIN             6		8
+FIGHTER_MAGE        6		8
+FIGHTER_CLERIC      6		8
+FIGHTER_THIEF       8		8
+FIGHTER_MAGE_THIEF  8		8
+DRUID               4		6
+RANGER              6		8
+MAGE_THIEF          8		6
+CLERIC_MAGE         6		6
+CLERIC_THIEF        8		6
+FIGHTER_DRUID       6		8
+FIGHTER_MAGE_CLERIC 6		8
+CLERIC_RANGER       6		8
+SORCERER            6		4
+MONK                4		6
+
+</pre>
+<br />
+
+{% capture note %}
+<ul>
+    <li>The listed values represent Dicesize (Dicenumber is hardcoded to 1).</li>
+    <li>Unlisted Classes end up using 1d6 for DEX and 1d8 for STR ("UNUSED" row).</li>
```

Comment:
Are you sure it's not 6 and 6, since that's the default table value? An unset class would match UNUSED.

Reply:
> Are you sure it's not 6 and 6, since that's the default table value? An unset class would match UNUSED.

Who knows?
The 0th row "UNUSED" specifies 8 for STR for whatever reason, maybe just copy/paste from another row...
Will check again when v2.6 drops...

Reply:
I suspect it's there just to make counting easier, like you note. It's the same in some other tables.

Reply:
Or to put it differently: they would have to specifically code such behaviour (unlisted being equal to 0).